backend/open_webui/utils/task.py

```
@@ -227,12 +227,12 @@ def rag_template(template: str, context: str, query: str):
 
 
 def title_generation_template(
-    template: str, messages: list[dict], user: Optional[Any] = None
+    template: str, gift_request_desc: str, messages: list[dict], user: Optional[Any] = None
```

Ruff (0.12.2)

Unused function argument: `messages`

(ARG001)